repos/system_upgrade/el7toel8/actors/initramfs/upgradeinitramfsgenerator/actor.py

```diff
@@ -0,0 +1,38 @@
+from leapp.actors import Actor
+from leapp.libraries.actor import upgradeinitramfsgenerator
+from leapp.models import (
+    BootContent,
+    RequiredUpgradeInitramPackages,  # deprecated
+    TargetUserSpaceInfo,
+    TargetUserSpaceUpgradeTasks,
+    UpgradeDracutModule,  # deprecated
+    UpgradeInitramfsTasks,
+    UsedTargetRepositories,
+)
+from leapp.tags import InterimPreparationPhaseTag, IPUWorkflowTag
+
+
+class UpgradeInitramfsGenerator(Actor):
+    """
+    Creates the upgrade initramfs
+
+    Creates an initram disk within a systemd-nspawn container using the target system userspace, including new kernel.
+    The creation of the initram disk can be influenced with RequiredUpgradeInitramPackages and UpgradeDracutModule,
```

Comment:
```suggestion
    The creation of the initram disk can be influenced with RequiredUpgradeInitramPackages and UpgradeDracutModule messages,
```

Don't we want to use the new models here?

Reply:
yep. fixed